.pre-commit-config.yaml

```diff
@@ -217,12 +217,14 @@ repos:
         files: \.py$
         exclude: ^tests/.*\.py$|^airflow/_vendor/.*$
         pass_filenames: true
+        require_serial: true
```

Comment:
Why do we want to `require_serial` ?

Reply:
Also this change belongs to [AIRFLOW-6004], right?

Reply:
Yep.